.ci/teardown.sh

```diff
@@ -81,13 +91,22 @@ collect_logs()
 
 		[ "${have_collect_script}" = "yes" ] && prefixes+=" ${collect_data_log_prefix}"
 
+		if [ "${have_procenv}" = "yes" ]
+		then
+			procenv --file "${procenv_user_log_path}"
+			sudo -E procenv --file "${procenv_root_log_path}"
+		fi
+
 		local prefix
 
 		for prefix in $prefixes
 		do
 			gzip -9 "$prefix"*
 		done
 
+		# The procenv log is tiny so doesn't require chunking
+		gzip -9 "${procenv_log_path}"
```

Comment:
I don't see where this variable was defined, I guess you mean `procenv_user_log_path` and/or `procenv_root_log_path`

Reply:
Aieeee! Good catch - branch updated ;)